stdlib/3/_winapi.pyi

```diff
@@ -59,23 +59,23 @@ def ConnectNamedPipe(handle: int, overlapped: Literal[False] = ...) -> None: ...
 @overload
 def ConnectNamedPipe(handle: int, overlapped: bool) -> Optional[Overlapped]: ...
 
-def CreateFile(file_name: str, desired_access: int, share_mode: int, security_attributes: int, creation_disposition: int, flags_and_attributes: int, template_file: int) -> int: ...
-def CreateJunction(src_path: str, dest_path: str) -> None: ...
-def CreateNamedPipe(name: str, open_mode: int, pipe_mode: int, max_instances: int, out_buffer_size: int, in_buffer_size: int, default_timeout: int, security_attributes: int) -> int: ...
-def CreatePipe(pipe_attrs: Any, size: int) -> Tuple[int, int]: ...
-def CreateProcess(application_name: Optional[str], command_line: Optional[str], proc_attrs: Any, thread_attrs: Any, inherit_handles: bool, creation_flags: int, env_mapping: Dict[str, str], cwd: Optional[str], startup_info: Any) -> Tuple[int, int, int, int]: ...
-def DuplicateHandle(source_process_handle: int, source_handle: int, target_process_handle: int, desired_access: int, inherit_handle: bool, options: int = ...) -> int: ...
-def ExitProcess(ExitCode: int) -> NoReturn: ...
+def CreateFile(__file_name: str, __desired_access: int, __share_mode: int, __security_attributes: int, __creation_disposition: int, __flags_and_attributes: int, __template_file: int) -> int: ...
+def CreateJunction(__src_path: str, __dst_path: str) -> None: ...
+def CreateNamedPipe(__name: str, __open_mode: int, __pipe_mode: int, __max_instances: int, __out_buffer_size: int, __in_buffer_size: int, __default_timeout: int, __security_attributes: int) -> int: ...
+def CreatePipe(__pipe_attrs: Any, __size: int) -> Tuple[int, int]: ...
+def CreateProcess(__application_name: Optional[str], __command_line: Optional[str], __proc_attrs: Any, __thread_attrs: Any, __inherit_handles: bool, __creation_flags: int, __env_mapping: Dict[str, str], __current_directory: Optional[str], __startup_info: Any) -> Tuple[int, int, int, int]: ...
+def DuplicateHandle(__source_process_handle: int, __source_handle: int, __target_process_handle: int, __desired_access: int, __inherit_handle: bool, __options: int = ...) -> int: ...
+def ExitProcess(__ExitCode: int) -> NoReturn: ...
 def GetACP() -> int: ...
 def GetFileType(handle: int) -> int: ...
```

Comment:
Just curious, is this one not positional-only?

Reply:
It isn't, for some reason. That one can be passed positionally or by name